_posts/2023-07-23-lawrence-gnd.md

```diff
@@ -0,0 +1,69 @@
+
+Then, several months later, Lawrence’s Parks & Rec department shocked the community by [spraying the remnant prairie](https://lawrencekstimes.com/2023/05/08/lprd-advisory-prairie-flowers/) behind the Prairie Park Nature Center with herbicide spray. Millions of flowers that had flourished in the prairie for millennia were destroyed. The city cited high turnover and the resulting loss of knowledge for how to care for the prairie.
+
+![Hoary Puccoon](/assets/img/hoary-puccoon1_full.jpg)
+*Hoary Puccoon, one species of flower present in the prairie before the spray*
+
+A loss of this magnitude is difficult to comprehend. There is little remaining of the prairie that used to stretch across the Midwest, and Lawrence’s lack of focus on how to take care of its outdoor spaces has permanently destroyed yet another part of this dwindling ecosystem. Many of the flowers lost in the spray were especially rare species, making this destruction even more devastating.
+
+There’s no guarantee something like this wouldn’t happen again, either – at least based on the way Lawrence operations have not considered sustainability and environmental consciousness “whenever possible”.
+
+## There’s Good News Too Though!
+
+I don’t mean to paint the city of Lawrence as a total climate justice villain – they aren’t, and there’s a lot of good they’re doing. The city and county are joining together to make a [Climate Action Plan](https://storymaps.arcgis.com/stories/22193694fa0544079c1f6a3de027aa90), though it was delayed a couple years due to COVID. Buses have been free to ride for [all of 2023](https://lawrencetransit.org/projects/fare-free/), part of a pilot project to see what kind of effect fare-free service would have on ridership. The Land Development Code, or zoning code, is [currently undergoing review](https://lawrenceldc.konveio.com/) that could encourage instead of discourage mixed-use developments and housing density. Both of these combined could lead to a much more walkable, bikeable, and therefore livable Lawrence. The city is launching a new [Homeless Programs Department](https://lawrencekstimes.com/2023/07/10/2024-draft-budget/) with a two-million-dollar budget. Actual action on housing everyone in Lawrence has been long, long overdue, and I’d advise everyone to keep an eye on this to make sure it isn’t just a shiny new version of the status quo. I’m hopeful that it will lead to real change, though. And, most recently, City Commission has passed an ordinance [declaring Lawrence as a safe haven](https://lawrencekstimes.com/2023/07/18/lawrencecitycomm-approves-ord9999/) for queer and gender non-conforming folks (and anyone facing any form of oppression), which is a huge win in a state with a legislature so hostile to its trans citizens.
```

Comment:
```suggestion
I don’t mean to paint the city of Lawrence as a total climate justice villain – they aren’t, and there’s a lot of good they’re doing. The city and county are joining together to make a [Climate Action Plan](https://storymaps.arcgis.com/stories/22193694fa0544079c1f6a3de027aa90), though it was delayed a couple years due to COVID. Buses have been free to ride for [all of 2023](https://lawrencetransit.org/projects/fare-free/), part of a pilot project to see what kind of effect fare-free service would have on ridership. The Land Development Code, or zoning code, is [currently undergoing review](https://lawrenceldc.konveio.com/) that would encourage mixed-use developments and housing density. Both of these combined could lead to a much more walkable, bikeable, and therefore livable Lawrence. The city is launching a new [Homeless Programs Department](https://lawrencekstimes.com/2023/07/10/2024-draft-budget/) with a two-million-dollar budget. Actual action on housing everyone in Lawrence has been long, long overdue, and I’d advise everyone to keep an eye on this to make sure it isn’t just a shiny new version of the status quo. I’m hopeful that it will lead to real change, though. And, most recently, City Commission has passed an ordinance [declaring Lawrence as a safe haven](https://lawrencekstimes.com/2023/07/18/lawrencecitycomm-approves-ord9999/) for queer and gender non-conforming folks (and anyone facing any form of oppression), which is a huge win in a state with a legislature so hostile to its trans citizens.
```